### PR TITLE
[FIX] Rename parameter small to tiny in XQuestResultXMLHandler

### LINKED ISSUE #2718: XQuestResultXMLHandler error on Windows.
Errors while compiling #2717 which means it probably fails on develop (without ASan or similar):

```
 4>C:\jenkins\ws\openms_pr_windows_ci\c2e226b2\source\src\openms\include\OpenMS/FORMAT/HANDLERS/XQuestResultXMLHandler.h(69): error C2144: syntax error: 'char' should be preceded by ')' [C:\jenkins\ws\openms_pr_windows_ci\c2e226b2\build\src\openms\OpenMS.vcxproj]

     4>C:\jenkins\ws\openms_pr_windows_ci\c2e226b2\source\src\openms\include\OpenMS/FORMAT/HANDLERS/XQuestResultXMLHandler.h(69): error C2144: syntax error: 'char' should be preceded by ';' [C:\jenkins\ws\openms_pr_windows_ci\c2e226b2\build\src\openms\OpenMS.vcxproj]

     4>C:\jenkins\ws\openms_pr_windows_ci\c2e226b2\source\src\openms\include\OpenMS/FORMAT/HANDLERS/XQuestResultXMLHandler.h(69): error C2059: syntax error: ')' [C:\jenkins\ws\openms_pr_windows_ci\c2e226b2\build\src\openms\OpenMS.vcxproj]

     4>C:\jenkins\ws\openms_pr_windows_ci\c2e226b2\source\src\openms\include\OpenMS/FORMAT/HANDLERS/XQuestResultXMLHandler.h(69): error C2208: 'char': no members defined using this type [C:\jenkins\ws\openms_pr_windows_ci\c2e226b2\build\src\openms\OpenMS.vcxproj]

     4>C:/jenkins/ws/openms_pr_windows_ci/c2e226b2/source/src/openms/source/FORMAT/HANDLERS/XQuestResultXMLHandler.cpp(57): error C2144: syntax error: 'char' should be preceded by ')' [C:\jenkins\ws\openms_pr_windows_ci\c2e226b2\build\src\openms\OpenMS.vcxproj]

     4>C:/jenkins/ws/openms_pr_windows_ci/c2e226b2/source/src/openms/source/FORMAT/HANDLERS/XQuestResultXMLHandler.cpp(57): error C2144: syntax error: 'char' should be preceded by ';' [C:\jenkins\ws\openms_pr_windows_ci\c2e226b2\build\src\openms\OpenMS.vcxproj]

     4>C:/jenkins/ws/openms_pr_windows_ci/c2e226b2/source/src/openms/source/FORMAT/HANDLERS/XQuestResultXMLHandler.cpp(57): error C2761: 'void OpenMS::Internal::XQuestResultXMLHandler::removeSubstring(OpenMS::String &,const OpenMS::String &)': member function redeclaration not allowed [C:\jenkins\ws\openms_pr_windows_ci\c2e226b2\build\src\openms\OpenMS.vcxproj]

     4>C:/jenkins/ws/openms_pr_windows_ci/c2e226b2/source/src/openms/source/FORMAT/HANDLERS/XQuestResultXMLHandler.cpp(57): error C2059: syntax error: ')' [C:\jenkins\ws\openms_pr_windows_ci\c2e226b2\build\src\openms\OpenMS.vcxproj]

     4>C:/jenkins/ws/openms_pr_windows_ci/c2e226b2/source/src/openms/source/FORMAT/HANDLERS/XQuestResultXMLHandler.cpp(58): warning C4091: '': ignored on left of 'char' when no variable is declared [C:\jenkins\ws\openms_pr_windows_ci\c2e226b2\build\src\openms\OpenMS.vcxproj]

     4>C:/jenkins/ws/openms_pr_windows_ci/c2e226b2/source/src/openms/source/FORMAT/HANDLERS/XQuestResultXMLHandler.cpp(58): error C2143: syntax error: missing ';' before '{' [C:\jenkins\ws\openms_pr_windows_ci\c2e226b2\build\src\openms\OpenMS.vcxproj]

     4>C:/jenkins/ws/openms_pr_windows_ci/c2e226b2/source/src/openms/source/FORMAT/HANDLERS/XQuestResultXMLHandler.cpp(58): error C2447: '{': missing function header (old-style formal list?) 
```

### PR DESCRIPTION
**Pull request by lukaszimmermann**
Fixes #2718

### CODE REVIEW

File: src/openms/include/OpenMS/FORMAT/HANDLERS/XQuestResultXMLHandler.h [hunk 1]
```diff
@@ -63,10 +63,10 @@ namespace OpenMS
 
       /**
        * @brief Removes a substring from a larger string
-       * @param large String from which `small` should be removed
-       * @param small Substring to be removed from `large`
+       * @param large String from which `tiny` should be removed
+       * @param tiny Substring to be removed from `large`
        */
-      static void removeSubstring(String & large, const String & small);
+      static void removeSubstring(String & large, const String & tiny);
```
Comment (cbielow) [inline src/openms/include/OpenMS/FORMAT/HANDLERS/XQuestResultXMLHandler.h]: would it make sense to move this into OpenMS::String() ?
And maybe add a while loop or document that only the first occurence is removed.

Comment (cbielow) [inline src/openms/include/OpenMS/FORMAT/HANDLERS/XQuestResultXMLHandler.h]: alternatively, you can remove this and just use `s.replace(small, "")`

Comment (lukaszimmermann) [inline src/openms/include/OpenMS/FORMAT/HANDLERS/XQuestResultXMLHandler.h]: Indeed. I do not really recall why I wrote that function in the first place.

Comment (timosachsenberg) [inline src/openms/include/OpenMS/FORMAT/HANDLERS/XQuestResultXMLHandler.h]: +1 for remove

**Comment by jpfeuffer** [comment]
Good. only LPWrapper_test fails. Merge?

**Pull request merged**